### ISSUE TITLE
Missing exchange config between France and Andorra

### ISSUE THREAD

**Issue by VIKTORVAV99** [electricitymaps/electricitymaps-contrib#7243]
## Description
There are a exchange between France and Andorra but we are missing this in the app.

The first step to get it added will be to create a exchange config for it.

**Comment by AJDelusion** [comment]
The connection from France to Jersey is also missing.

**Comment by VIKTORVAV99** [comment]
> The connection from France to Jersey is also missing.

Would you mind creating a new issue for that?

**Comment by jackkjenkins** [comment]
Hi @VIKTORVAV99 & ElectricityMaps Team,

I am interested in working on this issue. It's my first time contributing to this project, and I've had a look through closed PRs regarding config files and I think I understand how this would need to be implemented (in particular, I've looked at this Issue #7270 and PR #7298). 

Can I please be assigned to this issue, and is there anything else that I need to know before starting?

Thanks!

**Comment by VIKTORVAV99** [comment]
@jackkjenkins I've assigned you the issue and I don't think there is anything else you need to know if you have already looked through those issues.

**Comment by jackkjenkins** [comment]
Hi @VIKTORVAV99, 

I've added a PR for this issue. I wasn't able to add a reviewer so could you please add one for me if it is ready for review. 
If there's any issues please let me know. Thanks!

**Comment by consideRatio** [comment]
This issue is now resolved by @jackkjenkins via #7326 i think - close?
